packages/frontend/editor-ui/scripts/fetch-node-popularity.mjs

```diff
@@ -0,0 +1,95 @@
+#!/usr/bin/env node
+import { promises as fs } from 'fs';
+import path from 'path';
+import { fileURLToPath } from 'url';
+
+const __dirname = path.dirname(fileURLToPath(import.meta.url));
+const POPULARITY_ENDPOINT =
+	process.env.NODE_POPULARITY_ENDPOINT ||
+	'https://internal.users.n8n.cloud/webhook/nodes-popularity-scores';
```

Comment:
Not a fan of our internal instance URL so publicly shared. We could put it behind an alias or something. I see somewhere else in the code where we are setting this is not so important. 
Is the workflow retrieving this data resource intensive? Worth adding some caching?

Reply:
The data is pre-aggregated, so the workflow is not super resource intensive. It has week-long cache anyways using data tables.